docs/en/faq.md

```diff
@@ -66,11 +73,20 @@ Occasionally, the data processing job fails. You can re-run the failed job to re
 
 This solution uses a workflow named `ClickstreamLoadDataWorkflow`, orchestrated by Step Functions to load the processed data into Redshift. The workflow uses a DynamoDB table to record the files to be loaded that are processed by the data processing job. Any failure won't lose any data for loading into Redshift. It's safe to execute the workflow again to resume the loading workflow after it fails.
 
-## SDK
+### How do I recalculate historical events for out-of-the-box dashboards?
 
-### Can I use other SDK to send data to the pipeline created by this solution
+This solution only calculates the metrics in the out-of-the-box dashboards for the previous day. You might need to refresh the historical metrics after re-sending historical data, re-processing previous data, or re-importing data. You could manually reschedule the workflow to calculate historical data and update the existing metrics. Follow these steps:
```

Comment:
The metrics in the out-of-the-box dashboards are calculated on a daily basis. If you need to recalculate the metrics in case of there are changes to your historical data, you could manually reschedule the workflow to re-calculate the metrics. Follow these steps: